src/Csv.php

```diff
@@ -331,13 +332,35 @@ class Csv {
      * Constructor
      * Class constructor
      *
-     * @param  string|null  $input          The CSV string or a direct filepath
-     * @param  integer|null $offset         Number of rows to ignore from the beginning of  the data
-     * @param  integer|null $limit          Limits the number of returned rows to specified amount
-     * @param  string|null  $conditions     Basic SQL-like conditions for row matching
-     * @param  null|true    $keep_file_data Keep raw file data in memory after successful parsing (useful for debugging)
+     * @param  string|null $input The CSV string or a direct filepath
+     * @param  integer|null $offset Number of rows to ignore from the beginning
```

Comment:
The formatting seems to have changed without added benefit. I suspect you need to enable 'Align tag comments' in PhpStorm:

![image](https://user-images.githubusercontent.com/6819464/36640511-93cbd8ee-1a20-11e8-9de6-42855c67def4.png)

I've zipped a fresh copy of my `.idea` folder into #101. If you think that helps: [IntelliJ.and.PhpStorm.settings.zip](https://github.com/parsecsv/parsecsv-for-php/files/1755896/IntelliJ.and.PhpStorm.settings.zip)

Reply:
I didn't even know you could import the settings. I just replace the folder after comparing it with e.g. WinMerge. For other projects, we selectively commit handpicked files from the `.idea` folder, including the complete `codeStyles` folder. Everything else goes to `.gitignore` so no-one accidentally commits it.

Reply:
done